setup.py

```diff
@@ -11,7 +10,7 @@
 
 setup(
     name='stability-sdk',
-    version='0.2.7',
+    version='0.2.9',
```

Comment:
did we skip a version? in any event, I'd recommend only bumping the version immediately prior to release. I'm guessing the new version number here is from merging in updates from main which is fine, just wanted to highlight that I consider PyPI blocking publishing on version number conflicts a safety feature. Since the publish workflow triggers automatically, this is a tiny backstop in case we accidentally publish something prematurely. We should probably document this explicitly as part of the release process

Reply:
Yeah looks like 0.2.8 was released and shortly after 0.2.9 released which just changed `find_packages` to `find_namespace_packages` in the setup.py.